src/module.c

```diff
@@ -13401,7 +13404,10 @@ int RM_RdbSave(RedisModuleCtx *ctx, RedisModuleRdbStream *stream, int flags) {
     return REDISMODULE_OK;
 }
 
-/* Returns the internal secret of the cluster. */
+/* Returns the internal secret of the cluster.
+ * Should be used to authenticate as an internal connection to a shard in the
```

Comment:
in OSS these redis instances are called nodes
```suggestion
 * Should be used to authenticate as an internal connection to a node in the
```